xcp_d/utils/bids.py

```diff
@@ -361,6 +373,21 @@ def collect_run_data(layout, bold_file, cifti=False):
             suffix="xfm",
         )
 
+    elif not cifti:
+        run_data["boldref"] = layout.get_nearest(
+            bids_file.path,
+            strict=False,
+            suffix="boldref",
+        )
+        run_data["boldmask"] = layout.get(
+            return_type="file",
+            suffix="mask",
+            desc="brain",
+        )
+        run_data["t1w_to_native_xform"] = layout.get(
+            return_type="file", datatype="anat", suffix="xfm", to="MNI152NLin2009cAsym"
+        )
```

Comment:
@kahinimehta do you know why you had to add this part? I just realized that running xcp_d on HCP or DCAN data should automatically enable cifti processing, so AFAICT this shouldn't be reachable.

Reply:
I think this was supposed to address the "input_type not in ("hcp", "dcan)""  part of the loop

Reply:
This should catch when input_type is either "hcp" or "dcan", but is there any situation when we'd run nifti processing on hcp or dcan data?